docs/courses/challenge.md

```diff
@@ -0,0 +1,87 @@
+# LibreLingo Documentation on Challenge
+
+A **Challenge** is a single task or question the user responds to as they study using LibreLingo. Challenges are not configured directly, but [automatically generated from Skill configuration](skill.md#how-skills-generate-challenges).
+
+To get a better understanding, you can read the [course basics](README.md#basics).
```

Comment:
to get a better understanding of what?

Reply:
:laughing: I don't know, I think I copied this from another page and forgot to delete it.